docs/content/preview/yugabyte-cloud/managed-labs.md

```diff
@@ -11,22 +11,26 @@ menu:
 type: docs
 ---
 
-Use Product Labs to explore core features of YugabyteDB using a demo application connected to globally distributed test clusters in real time.
+Use Product Labs to explore core features of YugabyteDB using a demo application connected to globally distributed test clusters in real time. Labs run a live demo application, accessing real clusters deployed in a variety of geographies and topologies.
 
-The following lab is available (with more in development):
+To run a lab:
 
-- **Create Global Applications** - learn how you can minimize application latencies for users in widely dispersed geographies using [read replicas](../cloud-basics/create-clusters-topology/#read-replicas) and [partition by region](../cloud-basics/create-clusters-topology/#partition-by-region) deployments.
+1. [Sign up](https://cloud.yugabyte.com/signup?utm_medium=direct&utm_source=docs&utm_campaign=YBM_signup).
+1. [Log in](https://cloud.yugabyte.com/login).
+1. On the **Labs** page, choose a scenario and click **Try It Out Now** to launch the lab in a new tab.
+1. Follow the on-screen instructions.
 
-![Product Lab](/images/yb-cloud/managed-lab.png)
+    ![Run a Product Lab](/images/yb-cloud/managed-lab.png)
 
-## Run a lab
+## Labs
 
-Labs run a live demo application, accessing real clusters deployed in a variety of geographies and topologies.
+The following lab is available (with more in development).
```

Comment:
This statement seems unnecessary, unless you want to make sure to convey that there's more to come.